static/report/css/compare.css

```diff
@@ -0,0 +1,97 @@
+
+@import url("../../mfb/mfb.min.css");
+#chartContainer, #toolbar, #view, #templates {
+  margin: 0 5%;
+  width: 80%;
+  position: relative;
+  min-height: 1em;
+}
+
+#chartContainer svg {
+  width: 100%;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+
+.reticle .controls>span,
+.reticle .controls .begin,
+.reticle .controls .end {
+  display: block;
+  float: left;
+  margin: 0;
+
+}
+.reticle .controls .interstice {
+  width: 100%;
+  text-align: center;
+  line-height: 120%;
+  font-size: 200%;
+}
+.reticle .controls .begin {
+  position: absolute;
+  left: 0;
+}
+.reticle .controls .end {
+  position: absolute;
+  right: 0;
+}
+
+.reticle .controls INPUT {
+  line-height: 120%;
+  font-size: 200%;
+}
+.reticle .controls INPUT.end-input {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Element (INPUT.end-input) is overqualified, just use .end-input without element name.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2612918324/issues/source?bid=2452379&fileBranchId=2967311#l45)